documentation/modules/rn-2.5.adoc

```diff
@@ -280,6 +281,11 @@ In previous releases of {project-short}, VM with multiple disks that were migrat
 
 This issue is resolved in {project-short} {project-version}, VM with multiple disks that are migrated are able to boot on the target {ocp} cluster. link:https://issues.redhat.com/browse/MTV-433[(MTV-433)]
 
+
+.Transfer network not taken into account for cold migrations from vSphere
+In {project-short} releases 2.4.0-2.5.3, cold migrations from vSphere to the local cluster on which {project-short} was deployed did not take the transfer network into account. This issue is resolved in  {project-short} 2.5.4.  link:https://issues.redhat.com/browse/MTV-846[(MTV-846)]
```

Comment:
```suggestion
In {project-short} releases 2.4.0-2.5.3, cold migrations from vSphere to the local cluster on which {project-short} was deployed did not take a specified transfer network into account. This issue is resolved in  {project-short} 2.5.4.  link:https://issues.redhat.com/browse/MTV-846[(MTV-846)]
```
(the transfer network is optional, users can specify it if needed. if it's not specified the transfer is done over the pod network - and if there's no connectivity to vSphere from the pod network, that's going to be a problem)